### PR TITLE
added endpoint to get user by id throuigh directory service

### PR DESCRIPTION
**Pull request by elimelt**

### CODE REVIEW

File: server/directory/views.py [hunk 1]
```diff
@@ -21,4 +22,17 @@ def get(self, request):
...
         serializer = DirectoryMemberSerializer(members, many=True)
-        return Response(serializer.data)
\ No newline at end of file
+        return Response(serializer.data)
+
+class MemberDirectoryView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def get(self, request):
+        # get id from path parameter
```
Comment (ShawnCollinge) [inline server/directory/views.py:31]: nit: do you need this comment? The code itself shows what it does.

File: server/directory/views.py [hunk 2]
```diff
@@ -21,4 +22,17 @@ def get(self, request):
...
+        id = request.query_params.get('id')
+
+        # get Member object with member.user.id == id
```
Comment (ShawnCollinge) [inline server/directory/views.py:34]: nit: ditto with this comment. "Member.object.get(user_id)" is exactly what this comment says 😛

**Review by ShawnCollinge — APPROVED** [review]
LGTM other than the nitpicks.

File: server/directory/views.py (hunk 1 above)
Comment (elimelt) [reply server/directory/views.py:31]: I specifically wrote this because i wanted to differentiate between the id being from a query vs in the url itself.

Comment (ShawnCollinge) [reply server/directory/views.py:31]: I see, okay.

File: server/directory/views.py (hunk 2 above)
Comment (elimelt) [reply server/directory/views.py:34]: I despise the field lookup syntax and probably won't remember how it works a few months out of working with it.

Comment (elimelt) [reply server/directory/views.py:34]: however, if we end up writing a ton more views doing this i'll probably remove the comment eventually

**Pull request merged**